std_msgs/Header header
g_r[] lights